homeassistant/components/iota.py

```diff
@@ -0,0 +1,97 @@
+"""
```

Comment:
There is only a sensor. This doesn't require to include a component.

Reply:
Hey, not yet. I am going to add more features to this component. Is it necessary to be a sensor only PR?

Reply:
No, it's the right way to create a component if there is coming a binary sensor or switch platform which can share the common stuff. Otherwise everything can be handled in the platform itself as no sharing is needed.